### ISSUE TITLE
Can the licensing for javamelody be considered as ASL 2.0?

### ISSUE THREAD

**Issue by Cintha16** [javamelody/javamelody#883]
@evernat There was link on https://groups.google.com/forum/#!topic/javamelody/JI3yNH0Xmb0/discussion

Is this discussion concluded to consider it as ASL 2.0 only?

**Comment by evernat** [comment]
javamelody-core is licensed as ASL 2.0 as can be seen at https://github.com/javamelody/javamelody/blob/master/LICENSE or at https://repo1.maven.org/maven2/net/bull/javamelody/javamelody-core/1.80.0/javamelody-core-1.80.0.pom
javamelody-spring-boot-starter and the javamelody plugins for Jenkins, JIRA/Confluence/Bamboo/Bitbucket, Liferay, Sonarqube, Alfresco and Grails are also licensed as ASL 2.0.

But if you use one of the javamelody plugins for Jenkins or JIRA/Confluence/Bamboo/Bitbucket or Liferay or Sonarqube or Alfresco, they include by default the itext 2.1.7 library which is licensed as MPL 1.1 or LGPL, at your choice, as can be seen in the middle of http://search.maven.org/remotecontent?filepath=com/lowagie/itext/2.1.7/itext-2.1.7.pom.
So itext 2.1.7 is open-source and can be freely used and distributed in any software you want, including commercial projects.
(The license of itext 2.1.7 is different than the license of itext v5 or later)

When you use javamelody-core, javamelody-spring-boot-starter or one of the plugins, the jrobin library is also included in general (for the graphs). The license of jrobin is LGPL as can be seen at https://repo1.maven.org/maven2/org/jrobin/jrobin/1.5.9/jrobin-1.5.9.pom
So jrobin is also open-source and can be freely used and distributed in any software you want, including commercial projects. Not that LGPL is "Lesser GPL" and is not the same as GPL.

In conclusion, javamelody and its plugins use ASL 2.0 license and the javamelody dependencies use MPL or LGPL license. javamelody only depends on open-source with free software license, which can be freely used and distributed in any software you want, including commercial projects.
